src/amolMule/array2/ArrayOprns.java

```diff
@@ -0,0 +1,124 @@
+package amolMule.array2;
+
+/*
+ Write a method to return sum of all not prime numbers from given array.
+input : {11,13,17,22,44}
+output : 66
+Hint : (22+44 = 66)
+*/
+public class ArrayOprns {
+
+	int getSumNP(int[] arr) {
+		int s = 0;
+		for (int i = 0; i < arr.length; i++) {
+
+		}
+
+		return s;
+	}
+
+	boolean checkPrime(int n) {
+		boolean b = true;
+		int flag = 0;
```

Comment:
this flag variable not getting used in return or anywhere else, it's unused variable